File: src/models/Event.js
const mongoose = require("mongoose");

const Event = new mongoose.Schema({
  title: { type: String, required: true },
  content: { type: String, required: true },
  createdAt: { type: Object, required: true },
  authorID: { type: String, required: true },
});

module.exports = mongoose.model("Event", Event);

File: internal/models/event.go
package models

import "go.mongodb.org/mongo-driver/bson/primitive"

type Event struct {
	ID          primitive.ObjectID `json:"_id"          bson:"_id"`
	Title       string             `json:"title"        bson:"title"`
	Content     string             `json:"content"      bson:"content"`
	CreatedAt   string             `json:"createdAt"    bson:"createdAt"`
	AuthorLogin string             `json:"authorLogin"  bson:"authorLogin"`
}